stl/src/stacktrace.cpp

```diff
@@ -0,0 +1,345 @@
+                    const auto debug_create =
+                        reinterpret_cast<decltype(&DebugCreate)>(GetProcAddress(dbgeng, "DebugCreate"));
+
+                    // Deliberately not calling CoInitialize[Ex]. DbgEng.h API works fine without it.
+                    // COM initialization may have undesired interference with user's code.
+                    if (debug_create != nullptr
+                        && SUCCEEDED(debug_create(IID_IDebugClient, reinterpret_cast<void**>(&debug_client)))
+                        && SUCCEEDED(
+                            debug_client->QueryInterface(IID_IDebugSymbols, reinterpret_cast<void**>(&debug_symbols)))
+                        && SUCCEEDED(debug_client->QueryInterface(
+                            IID_IDebugControl, reinterpret_cast<void**>(&debug_control)))) {
+                        attached = SUCCEEDED(debug_client->AttachProcess(
+                            0, GetCurrentProcessId(), DEBUG_ATTACH_NONINVASIVE | DEBUG_ATTACH_NONINVASIVE_NO_SUSPEND));
+                        if (attached) {
+                            (void) debug_control->WaitForEvent(0, INFINITE);
+                        }
+
+                        constexpr ULONG add_options = 0x1 /* SYMOPT_CASE_INSENSITIVE */
+                                                    | 0x2 /* SYMOPT_UNDNAME */
+                                                    | 0x4 /* SYMOPT_DEFERRED_LOADS */
+                                                    | 0x10 /* SYMOPT_LOAD_LINES */
+                                                    | 0x20 /* SYMOPT_OMAP_FIND_NEAREST */
+                                                    | 0x100 /* SYMOPT_FAIL_CRITICAL_ERRORS */
+                                                    | 0x10000 /* SYMOPT_AUTO_PUBLICS */
+                                                    | 0x80000 /* SYMOPT_NO_PROMPTS */;
+
+                        constexpr ULONG remove_options = 0x8 /* SYMOPT_NO_CPP */
+                                                       | 0x40 /* SYMOPT_LOAD_ANYTHING */
+                                                       | 0x100 /* SYMOPT_NO_UNQUALIFIED_LOADS */
+                                                       | 0x400 /* SYMOPT_EXACT_SYMBOLS */
+                                                       | 0x1000 /* SYMOPT_IGNORE_NT_SYMPATH */
+                                                       | 0x4000 /* SYMOPT_PUBLICS_ONLY */
+                                                       | 0x8000 /* SYMOPT_NO_PUBLICS */
+                                                       | 0x20000 /* SYMOPT_NO_IMAGE_SEARCH */;
+
+                        (void) debug_symbols->AddSymbolOptions(add_options);
+                        (void) debug_symbols->RemoveSymbolOptions(remove_options);
+                    }
+                }
+            }
+
+            if (std::atexit(lock_and_uninitialize) != 0) {
+                uninitialize();
+                return false;
+            }

```

Comment:
We call `try_initialize()` repeatedly (once per `__std_stacktrace_meow()` call). This call to `std::atexit()` isn't guarded by `if (!initialize_attempted)`, so we'll repeatedly register `lock_and_uninitialize`. I believe that we'll eventually run out of slots in the atexit table, so this will fail.

Because `lock_and_uninitialize` is highly robust to partial initialization, I think that we can move the `std::atexit()` call within `if (!initialize_attempted)` very early, and then we wouldn't even need the `uninitialize()` call for `atexit` failure.

I will validate and push changes.